packages/commercetools/api-client/src/index.ts

```diff
@@ -29,33 +31,46 @@ let country = '';
 let countries = [];
 let currencies = [];
 let locales = [];
+let currentToken: Token = null;
+let api: ApiConfig = null;
+let tokenEvents: TokenEvents = {
+  onTokenChange: (token: Token) => {},
```

Comment:
also `on` indicates that something happened before and we are reacting to this event when in reality nothing happens. Shouldn't these methods be called just `changeToken` etc?